### DISCUSSION TITLE
Is it possible to get Postgres Error Code from Sea ORM `DbErr` ?

### DISCUSSION THREAD

**Discussion by leo91000** [SeaQL/sea-orm#773] [Q&A] [answered]
Hi !

Is there a way to retrive the postgres error code ([c.f postgres error codes](https://www.postgresql.org/docs/current/errcodes-appendix.html))  from a sea orm `DbErr(Query(String))` ?

Thanks

**Comment by ikrivosheev** [comment, marked as answer]
I think: https://github.com/SeaQL/sea-orm/pull/750 can help you in the next release.
